assets/js/src/form_editor/components/form_title.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from 'react';
+import classnames from 'classnames';
+
+export default () => {
+  const [isSelected, setIsSelected] = useState(false);
+  const titleClass = classnames('wp-block editor-post-title__block', {
+    'is-selected': isSelected,
+  });
+  return (
+    <div className="editor-post-title">
+      <div className={titleClass}>
+        <div>
+          <label htmlFor="post-title" className="screen-reader-text">
+            Add form name
+          </label>
+          <textarea
+            id="form-title"
+            className="editor-post-title__input"
+            placeholder="Add form name"
+            rows="1"
+            style={{
```

Comment:
I copypasted those from the component from Gutenberg. I'll check if they are really needed because our component is simpler than the one used in Gutenberg.